lib/ruby/lib/alces/stack/forkprocess.rb

```diff
@@ -0,0 +1,106 @@
+#==============================================================================
```

Comment:
So I'm having issues that metalware installs a separate copy of ruby that searches the stack for the ruby files. So their are two options for moving it:

1) Create a `Alces::Stack::Test` module that contains all test specific tests. Down side is that test code is contained with the production code

2) Add a new ruby path to somewhere in the test folder (however this proved to be difficult to achieve due to how ruby is installed)